e2e/util/selectors.ts

```diff
@@ -0,0 +1,32 @@
+import { Page } from '@playwright/test';
+
+class Selectors {
```

Comment:
Made this a class since selectors will _probably_ rely on each other in the future and they'd be easier to reference